docs/.readthedocs.yaml

```
@@ -0,0 +1,16 @@
+version: 2
+
+build:
+  os: ubuntu-20.04
+  tools:
+    python: "3.9"
```

From [the example](https://github.com/readthedocs-examples/example-sphinx-basic/blob/main/.readthedocs.yaml)
```suggestion
  os: ubuntu-22.04
  tools:
    python: "3.10"
```